Source/WebCore/accessibility/AXLogger.cpp

```diff
@@ -604,7 +604,8 @@ TextStream& operator<<(TextStream& stream, AXNotification notification)
     case AXNotification::name: \
         stream << #name; \
         break;
-    WEBCORE_AXNOTIFICATION_KEYS(WEBCORE_LOG_AXNOTIFICATION)
+
+        WEBCORE_AXNOTIFICATION_KEYS(WEBCORE_LOG_AXNOTIFICATION)
```

Comment:
This is borderline – `check-webkit-style` things this is something else than `case` / `default`, so expects it to be indented 4 spaces w.r.t. `switch` statement. However the macro expands to start with `case...`, so it "kind of" does start with `case` 😄.

I still obliged to get rid of the warning (and added extra newline before for better readability so that it does not visually blend with the macro definition as its another line). The other option was to put `// NOLINT` at the end of the `switch(...) {` line to silence the warning (it's not possible to put this comment on the line where the macro is used). Please let me know if you prefer that.